src/main/java/GenerationUtil.java

```diff
@@ -0,0 +1,15 @@
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class GenerationUtil {
+    public static final Integer DEFAULT_LIMIT = 1_000_000;
```

Comment:
```suggestion
    private static final Integer DEFAULT_LIMIT = 1_000_000;
```
if you don't user it outside the class, let's make it private